### PR TITLE
Fix ci assets

### PR DESCRIPTION
**Pull request by Saviq**
The neverputt failures are due to:
- canonical/mir#3647

### CODE REVIEW

File: mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot [hunk 1]
```diff
@@ -253,19 +255,40 @@ Slide Constraint Precedes Resize
...
-    Walk Pointer To ${BUTTON_CLOSE_FOCUSED}
+    Move Pointer To (0, 0)
+    ${pos}=                 Move Pointer To ${WINDOW_REGULAR_0_FOCUSED}
+    ${pos}=                 Displace ${pos} By (130, 0)
+    Move Pointer To ${pos}
+    Click LEFT Button
     Click LEFT Button
+    Wait Until Keyword Succeeds                     5                       1
+    ...                     Run Keyword And Expect Error                    ImageNotFoundError
+    ...                     VIDEO.Match             ${WINDOW_REGULAR_0_FOCUSED}                     0
```
Comment (hbatagelo) [inline mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot:271]:
```suggestion
    Wait Until ${WINDOW_REGULAR_0_FOCUSED} Is Absent
```

with the new keyword
```
Wait Until ${template} Is Absent
    Wait Until Keyword Succeeds    5    1
    ...    Run Keyword And Expect Error    ImageNotFoundError
    ...    VIDEO.Match    ${template}    0
```

File: mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot [hunk 2]
```diff
@@ -253,19 +255,40 @@ Slide Constraint Precedes Resize
...
+
+Close Floating Toplevel Window
+    Move Pointer To (0, 0)
+    ${pos}=                 Move Pointer To ${WINDOW_FLOATING_REGULAR_0_FOCUSED}
+    ${pos}=                 Displace ${pos} By (138, 0)
+    Move Pointer To ${pos}
+    Click LEFT Button
+    Wait Until Keyword Succeeds                     5                       1
+    ...                     Run Keyword And Expect Error                    ImageNotFoundError
+    ...                     VIDEO.Match             ${WINDOW_FLOATING_REGULAR_0_FOCUSED}            0
```
Comment (hbatagelo) [inline mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot:281]:
```suggestion
    Wait Until ${WINDOW_FLOATING_REGULAR_0_FOCUSED} Is Absent
```

File: mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot [hunk 3]
```diff
@@ -253,19 +255,40 @@ Slide Constraint Precedes Resize
...
-    ${pos}=                 Walk Pointer To ${WINDOW_DIALOG_FOCUSED}
+    Move Pointer To (0, 0)
+    ${pos}=                 Move Pointer To ${WINDOW_DIALOG_FOCUSED}
     ${pos}=                 Displace ${pos} By (112, 0)
-    Walk Pointer To ${pos}
+    Move Pointer To ${pos}
     Click LEFT Button
+    Wait Until Keyword Succeeds                     5                       1
+    ...                     Run Keyword And Expect Error                    ImageNotFoundError
+    ...                     VIDEO.Match             ${WINDOW_DIALOG_FOCUSED}                        0
```
Comment (hbatagelo) [inline mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot:291]:
```suggestion
    Wait Until ${WINDOW_DIALOG_FOCUSED} Is Absent
```

File: mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot [hunk 4]
```diff
@@ -253,19 +255,40 @@ Slide Constraint Precedes Resize
...
 
 *** Keywords ***
 Close Focused Toplevel Window
-    Walk Pointer To ${BUTTON_CLOSE_FOCUSED}
+    Move Pointer To (0, 0)
+    ${pos}=                 Move Pointer To ${WINDOW_REGULAR_0_FOCUSED}
+    ${pos}=                 Displace ${pos} By (130, 0)
+    Move Pointer To ${pos}
+    Click LEFT Button
     Click LEFT Button
```
Comment (hbatagelo) [inline mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot:268]: Is the double click really needed?

Comment (Saviq) [reply mir-ci/mir_ci/tests/robot/suites/mir_flutter_app/mir_flutter_app.robot]: Nope.

**Comment by Saviq** [comment]
>   ❌ test_scale[2.0-mir_demo_server]

This is a flaky test now, getting stuck (I think) getting the screenshot from the compositor. Something to look at separately.

**Pull request merged**